test/featureUtils_unit_test.js

```diff
@@ -0,0 +1,44 @@
+import GeoJSON2Feature from '../src/Renderer/ThreeExtended/GeoJSON2Feature';
+import FeaturesUtils from '../src/Renderer/ThreeExtended/FeaturesUtils';
+import Coordinates from '../src/Core/Geographic/Coordinates';
+/* global describe, it */
+
+const assert = require('assert');
+const geojson = require('../examples/geojson/simple.geojson.json');
+
+const feature = GeoJSON2Feature.parse('EPSG:4326', geojson, undefined, { buildExtent: true });
+
+describe('FeaturesUtils', function () {
+    it('should correctly parse geojson', (done) => {
+        assert.equal(feature.geometries.length, 3);
+        done();
```

Comment:
(nits: you don't need to call done or even have it in the parameter list in synchrone tests)